sdcm/cluster.py

```diff
@@ -2041,6 +2042,13 @@ def offline_install_scylla(self, unified_package, nonroot):
         """
         Offline install scylla by unified package.
         """
+        @retrying(n=30, sleep_time=15, allowed_exceptions=(UnexpectedExit, Libssh2_UnexpectedExit,))
+        def run_command_with_retry(command, sudo):
```

Comment:
it's weird we need this, doesn't our run command has retries parameter on it's own ?

Reply:
Yes, you are right. There is `retry` parameter of function